client/components/ReviewsWidget.jsx

```diff
@@ -16,24 +16,29 @@ export default class ReviewsWidget extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      reviewData: {
-        avgRating: 0,
-        itemName: '',
-        mainImage: '',
-        reviewsArray: [],
-      },
+      storeCount: 0,
+      rating: 0,
+      productCount: 0,
+      reviewList: [],
+      reviewPictures: [],
     };
   }
 
   componentDidMount() {
     // const queryString = window.location.pathname;
     // var id = url.substring(url.lastIndexOf('/') + 1);
     const id = 2;
-    axios.get(`/reviews/all/${id}`)
-      .then((response) => {
+    Promise.all([
+      axios.get(`/review-summary/${id}`),
+      axios.get(`/review-list/${id}`),
+      axios.get(`/reviews-pictures/${id}`),
+    ])
+      .then(([reviewSummary, reviewList, reviewPictures]) => {
         this.setState({
-          reviewData: response.data,
-        });
+          ...reviewSummary.data,
```

Comment:
It's not clear to me why you're handling this different than the two below so I thought I'd point it out just in case.

Reply:
ReviewSummary gets destructured to the state (main level: storeCount, rating, productCount), but reviewList are reviewPictures are arrays, they meant to replace reviewList and reviewPictures arrays in the state, so therefore should not be destructured into the state.